thirdparty/lua-Spore/form-data-filename.patch

```diff
@@ -0,0 +1,28 @@
+From 912b0235b61c77b019dec42a27c349c761b06517 Mon Sep 17 00:00:00 2001
```

Comment:
I forgot to say, but all the patch really needs starts from line 10. So it's probably best to limit it to that.